docs/reference/modules/snapshots.asciidoc

```diff
@@ -1,39 +1,52 @@
 [[modules-snapshots]]
 == Snapshot And Restore
 
-The snapshot and restore module allows to create snapshots of individual
-indices or an entire cluster into a remote repository like shared file system,
-S3, or HDFS. These snapshots are great for backups because they can be restored
-relatively quickly but they are not archival because they can only be restored
-to versions of Elasticsearch that can read the index. That means that:
+You can store snapshots of individual indices or an entire cluster in
+a remote repository like a shared file system, S3, or HDFS. These snapshots
+are great for backups because they can be restored relatively quickly. However,
+snapshots can only be restored to versions of Elasticsearch that can read the
+index:
```

Comment:
"read the index" to "read the index version within the snapshot"?

Reply:
It's complicated at the moment. Basically, an index can migrate from one version to another and contain segments from older versions. It can also contain settings that are not allowed in the new version but were allowed for the index that got migrated. So, to keep things simple the emphasis here should be on the version where index was created and not snapshotted as we have in the table below.